packages/host/app/components/operator-mode/workspace-chooser/index.gts

```diff
@@ -31,6 +34,9 @@ export default class WorkspaceChooser extends Component<Signature> {
               data-test-workspace={{realmURL}}
             />
           {{/each}}
+          {{#if this.matrixService.initializeNewUserRealm.isRunning}}
```

Comment:
just set a tracked property in the service at the beginning and end of the promise instead of converting to a task. a task is really not necessary for a service because it is longed lived.

Reply:
Updated